docs/user/tutorial.rst

```diff
@@ -1295,10 +1295,199 @@ Inspecting the application now returns:
         ⇒ /images/{name} - Item:
            └── GET - on_get
 
-.. Query Strings
-.. -------------
+Query Strings
+-------------
+Now that we are able to get the images from the service, we need a way to get a list available images. We have already set up this route. Before testing this route let's change its output format back to JSON to have a more terminal-friendly output. The top of file ``images.py`` should look like this:
+
+.. code:: python
+
+    import io
+    import os
+    import re
+    import uuid
+    import mimetypes
+
+    import falcon
+    import json
+
+
+    class Collection:
+
+        def __init__(self, image_store):
+            self._image_store = image_store
+
+        def on_get(self, req, resp):
+            max_size = int(req.params.get("maxsize", 0))
+            images = self._image_store.list(max_size)
+            doc = {
+                'images': [
+                    {'href': '/images/'+image} for image in images
+            ]
+        }
+
+        resp.text = json.dumps(doc, ensure_ascii=False)
+        resp.status = falcon.HTTP_200
+
+    def on_post(self, req, resp):
+        name = self._image_store.save(req.stream, req.content_type)
+        resp.status = falcon.HTTP_201
+        resp.location = '/images/' + name
+
+
+Now try the following:
+
+.. code:: bash
+
+    http localhost:8000/images
+
+In response you should get the following data that we statically have put in the code.
+
+.. code::
+
+    {
+        "images": [
+            {
+                "href": "/images/1eaf6ef1-7f2d-4ecc-a8d5-6e8adba7cc0e.png"
+            }
+        ]
+    }
+
+Let's go back to the ``on_get`` method and create a dynamic response. We can use query strings to set maximum image size and get the list of all images smaller than the specified value.
+
+.. code:: python
+
+    import io
+    import os
+    import re
+    import uuid
+    import mimetypes
+
+    import falcon
+    import json
+
+
+    class Collection:
+
+        def __init__(self, image_store):
+            self._image_store = image_store
+
+        def on_get(self, req, resp):
+            max_size = int(req.params.get("maxsize", 0))
```

Comment:
I think it would be useful to showcase `get_param_as_int` here, so we could do
```suggestion
            max_size = int(req.get_param_as_int("maxsize", default=0, min_value=1))
```
(I've added a min value but we may omit it too)